### PR TITLE
Added Docs for LiveQuery Options

### PR DESCRIPTION
**Pull request by AliabbasMerchant**
@YourTechBud Done

### CODE REVIEW

File: docs/manual/database/live-query.md [hunk 1]
```diff
@@ -132,13 +142,161 @@ func main() {
...
 - **delete** - Whenever any data is deleted
+**changedDoc:** The doc that changed.
 
 `onError` function is called with the `error` if there was any error subscribing to data.
 
 As you would have noticed the `subscribe` function returns an `unsubscribe` function. You should call this function whenever you want to unsubscribe to the changes.
 
+## Setting the liveQuery options:
+You can set the liveQuery options using the `options()` function.  
+The function helps to set `changesOnly` to true or false(default).
```
Comment (YourTechBud) [inline docs/manual/database/live-query.md]: Space required after the false

File: docs/manual/database/live-query.md [hunk 2]
```diff
@@ -132,13 +142,161 @@ func main() {
...
+      <li class="tab col s2"><a class="active" href="#live-query-options-js">Javascript</a></li>
+      <li class="tab col s2"><a href="#live-query-options-java">Java</a></li>
+      <li class="tab col s2"><a href="#live-query-options-python">Python</a></li>
+      <li class="tab col s2"><a href="#live-query-options-golang">Golang</a></li>
+    </ul>
+  </div>
+  <div id="live-query-options-js" class="col s12" style="padding:0">
+    <pre>
+      <code>
+import { API, cond, or, and } from 'space-api';
```
Comment (YourTechBud) [inline docs/manual/database/live-query.md]: In the examples we just need to have the code concerning the live query. The imports and error handling isnt required

Comment (AliabbasMerchant) [reply docs/manual/database/live-query.md]: We have nearly always included whole code snippets, including imorts and error handling

Comment (YourTechBud) [reply docs/manual/database/live-query.md]: Have a look at read data js docs.

**Comment by AliabbasMerchant** [comment]
@YourTechBud  Done

**Pull request merged**